src/Microsoft.Framework.FileSystemGlobbing/Abstractions/DirectoryInfoWrapper.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.Framework.FileSystemGlobbing.Abstractions
+{
+    public class DirectoryInfoWrapper : DirectoryInfoBase
+    {
+        private DirectoryInfo DirectoryInfo;
+
+        public DirectoryInfoWrapper(DirectoryInfo directoryInfo)
+        {
+            DirectoryInfo = directoryInfo;
+        }
+
+        public override IEnumerable<FileSystemInfoBase> EnumerateFileSystemInfos(string searchPattern, SearchOption searchOption)
```

Comment:
We are not going to this to avoid a reverse dependency from Microsoft.Framework.* to Microsoft.AspNet.*.